development-tools/clinic.rst

```diff
@@ -1996,6 +1996,36 @@ The generated glue code looks like this:
       return return_value;
    }
 
+You can lock up to two additional objects
```

Comment:
```suggestion
You can lock one or two additional objects
```
Maybe this is clearer since I initially read it as "(lock up) to" instead of "lock (up to)".

Reply:
Thanks, that's less confusing!